.github/workflows/build.yml

```diff
@@ -15,17 +15,13 @@ jobs:
     - name: Validate podspec
       run: pod lib lint
 
-  build:
-    runs-on: macos-11
+  build_xcode14:
+    runs-on: macos-12
     strategy:
       matrix:
         run-config:
-          - { xcode_version: '11.7', simulator: 'name=iPhone SE (2nd generation),OS=13.7' }
-          - { xcode_version: '12.5.1', simulator: 'name=iPhone SE (2nd generation),OS=14.5' }
-          - { xcode_version: '13.0', simulator: 'name=iPad Pro (12.9-inch) (5th generation),OS=15.0' }
-
-
-    steps:
+          - { xcode_version: '14.1', simulator: 'name=iPad Pro (12.9-inch) (5th generation),OS=16.1' }
+    steps: &build_steps
```

Comment:
Bummer... I liked the cleanup with the anchor, but GitHub doesn't like it:

```
The workflow is not valid. .github/workflows/build.yml: Anchors are not currently supported. Remove the anchor 'build_steps'
```

Reply:
My bad. I will fix it and push it right now!